llvm/test/Transforms/LoopVectorize/RISCV/branch-frequency-cost.ll

```diff
@@ -0,0 +1,113 @@
+; RUN: opt -mtriple=riscv64 -mattr=+v -riscv-v-vector-bits-max=128 -riscv-v-register-bit-width-lmul=1 -passes='require<profile-summary>,loop-vectorize' -debug-only=loop-vectorize -disable-output < %s 2>&1 | FileCheck %s
+
+; Check that branch weights make a difference when computing cost of scalar loop
+
+define void @foo_with_wts(ptr %A, ptr %B, i32 %n) {
+; CHECK: LV: Checking a loop in 'foo_with_wts'
+; CHECK: LV: Scalar loop costs: [[COST:[0-9]+]].
```

Comment:
This is a rare case where I think it would make sense to check the costs as printed in the debug output to make sure they are scaled as expected